frontend/src/components/navigation/MenuItem.tsx

```diff
@@ -0,0 +1,41 @@
+import { Menu } from 'antd';
+import { Link } from 'react-router-dom';
+import styled from 'styled-components';
+import { Colors } from '../../styles';
+
+const Container = styled.div`
+
+  background-color:inherit;
+  color: ${Colors.grey};
```

Comment:
`grey` to niezbyt dobra nazwa dla reużywalnego koloru. Powinno być tutaj coś typu `Colors.sider`. Co jeśli zmienilibyśmy kolor np. na czerwony? Wtedy w każdym miejscu gdzie mieliśmy `Colors.grey` musielibyśmy zmienić na `Colors.red`, bo pod spodem grey nie może siedzieć czerwony.

Dzięki dodaniu bardziej ogólnych nazw, znika nam problem z jakąkolwiek modyfikacją wyglądu w przyszłosci.